api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -447,6 +447,11 @@ type NonAdmin struct {
 	// which restore spec field values to enforce
 	// +optional
 	EnforceRestoreSpec *velero.RestoreSpec `json:"enforceRestoreSpec,omitempty"`
+
+	// GarbageCollectionPeriod defines how frequently to look for possible leftover non admin objects in OADP namespace.
+	// By default 5m
```

Comment:
the default will be applied in NAC side, but documentation will live here

please tell me what should be the default, I just added a random value

Reply:
The scenario where residual Velero objects get leftover won't be very frequent, right ? 
I think daily cleanup as a default (24 hr) would be reasonable here.

Reply:
will change doc here and value in code in NAC side

Reply:
Probably matching one of these times
https://github.com/search?q=repo%3Avmware-tanzu%2Fvelero+time.Minute+*+%3D+NOT+path%3Atest.go&type=code

Reply:
24 hrs works

Reply:
That should discourage frequent nonAdminSchedules that we sometimes see admins do.. hourly backup of entire clusters.. 😮‍💨